### ISSUE TITLE
Disable scrolling

### ISSUE THREAD

**Issue by kevinlu1211** [sohobloo/react-native-modal-dropdown#178]
I was wondering if I could disable scrolling for my modal dropdown?

**Comment by oschvr** [comment]
Try `scrollEnabled={false}`

**Comment by isinha-onefc** [comment]
> Try `scrollEnabled={false}`

No, I would rather stop the initial scrolling of the list to the selected index.
